SWEPortal/SWEPortal/ClientApp/src/components/views/Sessions.js

```diff
@@ -1,73 +1,246 @@
 import { useState, useEffect } from 'react'
 import './base.css'
 import { EntryTable, FormHeader } from './Shared.js'
+import { AiFillEdit, AiFillDelete } from 'react-icons/ai';
+import apiBuilder from '../../api/rooms'
+
+const api = apiBuilder("sessions")
+const roomApi = apiBuilder("rooms")
+const speakerApi = apiBuilder("speakers")
+const timeSlotApi = apiBuilder("timeslots")
```

Comment:
since these are used more than once there should probably be an intermediate class like roomsApi() that would create the apiBuilder("sessions"). Dont need to do it if you dont want to.

Reply:
Yeah ultimately the code isn't awesome. I will take the time to refactor once it is all working.